ccvm_simulators/solvers/dl_solver.py

```diff
@@ -298,12 +423,14 @@ def _solve(
                 * np.sqrt(dt)
                 / noise_ratio_i
             )
-            
-            diff = 2 * g * torch.sqrt(c**2 + s**2 + 0.5) 
-            
-            c += dt * c_drift + diff * wiener_increment_c
-            
-            s += dt * s_drift + diff * wiener_increment_s
+            c += (
+                dt * c_drift
+                + 2 * g * torch.sqrt(c**2 + s**2 + 0.5) * wiener_increment_c
+            )
+            s += (
+                dt * s_drift
+                + 2 * g * torch.sqrt(c**2 + s**2 + 0.5) * wiener_increment_s
+            )
```

Comment:
I think this may have been a mistakenly-resolved merge conflict; please revert this section back to Mehmet's code with the `diff` variable

Reply:
You are right. I've fixed them. Thanks Katie!